docs/Users_Guide/glossary.rst

```diff
@@ -13319,6 +13269,772 @@ METplus Configuration Glossary
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_MEAN_REGRID_METHOD
+     Specify the value for 'climo_mean.regrid.method' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_MEAN_REGRID_WIDTH
+     Specify the value for 'climo_mean.regrid.width' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_MEAN_REGRID_VLD_THRESH
+     Specify the value for 'climo_mean.regrid.vld_thresh' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_MEAN_REGRID_SHAPE
+     Specify the value for 'climo_mean.regrid.shape' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_FILE_NAME
+     Specify the value for 'climo_stdev.file_name' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_FIELD
+     Specify the value for 'climo_stdev.field' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_TIME_INTERP_METHOD
+     Specify the value for 'climo_stdev.time_interp_method' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_DAY_INTERVAL
+     Specify the value for 'climo_stdev.day_interval' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_HOUR_INTERVAL
+     Specify the value for 'climo_stdev.hour_interval' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_REGRID_METHOD
+     Specify the value for 'climo_stdev.regrid.method' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_REGRID_WIDTH
+     Specify the value for 'climo_stdev.regrid.width' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_REGRID_VLD_THRESH
+     Specify the value for 'climo_stdev.regrid.vld_thresh' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_STDEV_REGRID_SHAPE
+     Specify the value for 'climo_stdev.regrid.shape' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_CDF_CDF_BINS
+     See :term:`PAIR_STAT_CLIMO_CDF_BINS`
+
+   PAIR_STAT_CLIMO_CDF_BINS
+     Specify the value for 'climo_cdf.cdf_bins' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_CDF_CENTER_BINS
+     Specify the value for 'climo_cdf.center_bins' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_CDF_WRITE_BINS
+     Specify the value for 'climo_cdf.write_bins' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_CLIMO_CDF_DIRECT_PROB
+     Specify the value for 'climo_cdf.direct_prob' in the MET configuration file for PairStat.
+
+     | *Used by:* PairStat
+
+   PAIR_STAT_OBS_WINDOW_BEG
```

Comment:
Side note, the obs window is being parsed from the Pair-Stat configuration file but not actually USED in the initial version to filter pairs by time yet. Good to have in the wrapper for now, but there's potential for this to changes as the tool matures.